src/Routes.tsx

```diff
@@ -256,8 +256,8 @@ class Routes extends React.Component<WithStyles<typeof styles>, { openDrawer: bo
                 target="_blank"
                 rel="noopener noreferrer"
               >
-                <Icon className={classNames('fa', 'fa-github', classes.leftIcon)} />
-                <span className="d-none d-md-block">GitHub</span>
+                <Icon className={classNames('fa', 'fa-code', classes.leftIcon)} />
+                <span className="d-none d-md-block">Source</span>
```

Comment:
Let's keep it GitHub. The other icon doesn't have the label just a tooltip.

Reply:
Still, I think this just looks better.

Reply:
I don't have strong opinion about it. 🤷‍♂️

Reply:
Thanks for accepting it.